_episodes/07-thresholding.md

```diff
@@ -261,47 +217,42 @@ colored shapes from the original, as shown in this image:
 > >  *
 > >  * usage: python ThresholdPractice.py <filename> <kernel-size> <threshold>
 > > '''
-> > import cv2, sys
+> > import sys
+> > import numpy as np
+> > import skimage.color
+> > import skimage.fiters
+> > import skimage.io
+> > import skimage.viewer
 > > 
 > > # get filename, kernel size, and threshold value from command line
 > > filename = sys.argv[1]
 > > k = int(sys.argv[2])
 > > t = int(sys.argv[3])
```

Comment:
should change to either `T = float(sys.argv[3]) / 255` and expect a number between 0..255, or just `T = float(sys.argv[3])`